chains/orchestrator-relays/runtime/starlight/constants/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod weights;
+
+/// Money matters.
+pub mod currency {
+    use primitives::Balance;
+
+    /// The existential deposit.
+    pub const EXISTENTIAL_DEPOSIT: Balance = 1 * CENTS;
+
+    // Provide a common factor between runtimes based on a supply of 10_000_000 tokens.
+    pub const SUPPLY_FACTOR: Balance = 100;
+
+    pub const UNITS: Balance = 1_000_000_000_000;
+    pub const CENTS: Balance = UNITS / 30_000;
+    pub const GRAND: Balance = CENTS * 100_000;
+    pub const MILLICENTS: Balance = CENTS / 1_000;
+    pub const MICROUNITS: Balance = 1_000_000;
+    pub const MILLIUNITS: Balance = 1_000_000_000;
+
+    pub const STORAGE_BYTE_FEE: Balance = 100 * MICROUNITS * SUPPLY_FACTOR;
+    pub const STORAGE_ITEM_FEE: Balance = 100 * MILLIUNITS * SUPPLY_FACTOR;
+
+    pub const fn deposit(items: u32, bytes: u32) -> Balance {
+        items as Balance * STORAGE_ITEM_FEE + (bytes as Balance) * STORAGE_BYTE_FEE
+    }
+}
+
+/// Time and blocks.
+pub mod time {
+    use primitives::{BlockNumber, Moment};
+    pub const MILLISECS_PER_BLOCK: Moment = 6000;
+    pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
+
+    tp_traits::prod_or_fast_parameter_types! {
+        pub const EpochDurationInBlocks: BlockNumber = { prod: 6 * HOURS, fast: 1 * MINUTES };
+    }
+
+    // These time units are defined in number of blocks.
+    pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+    pub const HOURS: BlockNumber = MINUTES * 60;
+    pub const DAYS: BlockNumber = HOURS * 24;
+    pub const WEEKS: BlockNumber = DAYS * 7;
+
+    // 1 in 4 blocks (on average, not counting collisions) will be primary babe blocks.
+    // The choice of is done in accordance to the slot duration and expected target
+    // block time, for safely resisting network delays of maximum two seconds.
+    // <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
+    pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
+}
+
+pub mod snowbridge {
+    use {
+        frame_support::parameter_types,
+        xcm::prelude::{Location, NetworkId},
+    };
+
+    parameter_types! {
+            /// Network and location for the Ethereum chain. On Stagelight, the Ethereum chain bridged
+            /// to is the Sepolia Ethereum testnet, with chain ID 11155111.
+            /// <https://chainlist.org/chain/11155111>
+            /// <https://ethereum.org/en/developers/docs/apis/json-rpc/#net_version>
+            pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 11155111 };
```

Comment:
this one is wrong, we need to put the mainnet one, not the sepolia one (1)

Reply:
fixed